.circleci/config.yml

```diff
@@ -144,10 +144,9 @@ jobs:
       - run:
           apt-get install -y --no-install-recommends
             ca-certificates
-            clang-tidy-12
+            clang-tidy
```

Comment:
I dont think this is a good idea. different clang-format/-tidy versions are not super compatible.

Reply:
Done, changed to -14.